swan-lake/learn-the-platform/run-in-the-cloud/function-as-a-service/aws-lambda.md

```diff
@@ -1,193 +1,106 @@
 ---
 layout: ballerina-cloud-left-nav-pages-swanlake
 title: AWS Lambda
-description: Learn how to write and deploy AWS Lambda functions using ballerina
-keywords: ballerina, programming language, serverless, cloud, aws, lambda, cloud native
+description: AWS Lambda is an event-driven, serverless computing platform. The AWS Lambda extension provides the functionality to write AWS Lambda-compatible packages by exposing a Ballerina function as an AWS Lambda function.
+keywords: ballerina, programming language, serverless, cloud, aws, lambda, cloud-native
 permalink: /learn/run-ballerina-programs-in-the-cloud/function-as-a-service-with-ballerina/aws-lambda/
 active: aws-lambda
-intro: The AWS Lambda extension provides the functionality to expose a Ballerina function as an AWS Lambda function.
+intro: AWS Lambda is an event-driven, serverless computing platform. The AWS Lambda extension provides the functionality to write AWS Lambda-compatible packages by exposing a Ballerina function as an AWS Lambda function.
 ---
 
-## Prerequisites
-* Install the latest Ballerina distribution.
-* Install the <a href="https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html" target="_blank">AWS CLI</a>.
-* Go to <a href="https://console.aws.amazon.com/iam/home?#/users" target="_blank">users</a>, and click **Add User**.
-* Enter the username, enable programmatic access, and make sure the user has the `AWSLambda_FullAccess` or higher permissions.
-* Configure the <a href="https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html" target="_blank">AWS CLI</a> using the access key and secret generated in the user creation.
-* Go to <a href="https://console.aws.amazon.com/iamv2/home#/roles" target="_blank">roles</a>, and create a role that has the `AWSLambdaBasicExecutionRole` or higher permissions.
-* Go to the newly created role, and copy the role ARN to use when the Lambda function is being deployed.
+The AWS Lambda extension provides the functionality to expose a Ballerina function as an AWS Lambda function. Ballerina functions can be deployed in AWS Lambda by annotating a Ballerina function with `@awslambda:Function`. 
+
+> **Note:** An AWS Lambda function always adheres to the following function signature: 
+`function (awslambda:Context, json|EventType) returns json|error`
```

Comment:
Shouldn't we have code line in a new line? 
Due to some reason GH preview is showing it on the same line. 
<img width="890" alt="image" src="https://github.com/ballerina-platform/ballerina-dev-website/assets/16300038/a1237e4c-6eeb-4a8b-bbb8-27ed59dcbd91">

Reply:
Added a `<br/>` tag to get the code to a new line as shown below.

<img width="970" alt="Screenshot 2023-07-13 at 05 16 34" src="https://github.com/ballerina-platform/ballerina-dev-website/assets/11707273/b751fa73-f2ee-4421-8023-443c5fa745c6">